pages/_app.tsx

```diff
@@ -50,6 +51,32 @@ export default function App({ Component, pageProps }: AppProps): JSX.Element {
           `,
         }}
       />
+      <DefaultSeo
+        title="Mutual Aid NYC"
+        description="Our community-sourced, volunteer-curated library is a growing collection of the many resources available to New Yorkers. Mutual Aid NYC is committed to building a comprehensive list of high-quality resources—check back frequently, as new resources are added every day!"
+        canonical="https://lists.mutualaid.nyc/"
```

Comment:
Maybe for another MR but we should add a like `ENV` env variable (e.g. local, staging, etc) and determine urls based off of that instead of hard-coding the prod one.

Could also see that being overkill since theoretically we'd only be sharing prod links.

Reply:
Id be down to do that in a different PR, if you want but I do agree that its a bit overkill since we're only sharing on prod and we can't really test OpenGraph links with a localhost url

Reply:
It _could_ be nice to test in staging but let's just leave it for now